indigo/analysis/italics_terms.py

```diff
@@ -10,19 +10,23 @@ class BaseItalicsFinder(LocaleBasedMatcher, TextPatternMarker):
     """ Italicises terms in a document.
     """
     marker_tag = 'i'
-    candidate_xpath = ".//text()[not(ancestor::a:i)]"
 
     def mark_up_italics_in_document(self, document, italics_terms):
         """ Find and italicise terms in +document+, which is an Indigo Document object.
         """
         # we need to use etree, not objectify, so we can't use document.doc.root,
         # we have to re-parse it
         root = etree.fromstring(document.content)
+        self.get_candidate_xpath(italics_terms)
         self.setup(root)
         self.get_pattern_re(italics_terms)
         self.markup_patterns(root)
         document.content = etree.tostring(root, encoding='utf-8').decode('utf-8')
 
+    def get_candidate_xpath(self, terms):
+        xpath_contains = " or ".join([f"contains(., '{term}')" for term in terms])
+        self.candidate_xpath = f".//text()[{xpath_contains} and not(ancestor::a:i)]"
```

Comment:
I think you'll want brackets around the contains clause, so that it's `(A or B or C) and not(ancestor::a:i)` otherwise the boolean logic precedence is incorrect (it'll be `A or B or (C and not(...))` because `and` binds more tightly than `or`).